VidCloud9/dist/presence.js

```diff
@@ -0,0 +1,125 @@
+var presence = new Presence({
+    clientId: "697552926876368917"
+}),
+
+strings = presence.getStrings({
+    play: "presence.playback.playing",
+    pause: "presence.playback.paused"
+});
+var iFrameVideo, currentTime, duration, paused;
+var video, videoDuration, videoCurrentTime;
+var lastPlaybackState = null;
+var playback;
+var browsingStamp = Math.floor(Date.now() / 1000);
+var title;
+
+presence.on("iFrameData", (data) => {
+    video = data;
+    playback = data.iframe_video.duration !== null ? true : false;
+    if (playback) {
+        iFrameVideo = data.iframe_video.iFrameVideo;
+        currentTime = data.iframe_video.currTime;
+        duration = data.iframe_video.dur;
+        paused = data.iframe_video.paused;
+    }
+    video = data;
+});
+
+if (lastPlaybackState != playback) {
+    lastPlaybackState = playback;
+    browsingStamp = Math.floor(Date.now() / 1000);
+}
+
+presence.on("UpdateData", async() => {
+    var timestamps = getTimestamps(Math.floor(currentTime), Math.floor(duration));
+    var presenceData = {
+        largeImageKey: "logo", 
+    };
+    if(document.location.pathname == "/"){
+        presenceData.startTimestamp = browsingStamp;
+        presenceData.details = "Viewing home page";
+    }
+    else if(document.location.pathname == "/movies"){
```

Comment:
Use `switch` instead of this. Check this for more information: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch